main/distributed-programming.md

```diff
@@ -1,47 +1,55 @@
-# Distributed programming in JavaScript
+# Agoric JavaScript Programming Extensions
 
-Agoric's platform is built mainly in JavaScript and enables authors to write secure smart contracts in JavaScript.
+Agoric's platform lets you write secure smart contracts in JavaScript. The platform itself is mainly written in JavaScript. However, we've made several Agoric-specific additions to general JavaScript programming that you should know about and understand before programming on the platform. Some are *concepts*, others are *Agoric library additions*, and some are at the *syntax level*.
```

Comment:
I'd add that all changes at the language level are in process to becoming official standards.